### PR TITLE
Add writings from Cards Against Humanity

### LINKED ISSUE #37: Retrieve Cards Against Humanity Writings
Add the writing from [Cards Against Humanity](https://www.cardsagainsthumanity.com) in accordance with the [Creative Commons BY-NC-SA 4.0 licence](https://creativecommons.org/licenses/by-nc-sa/4.0/)

### PR DESCRIPTION
**Pull request by taitfuller**
Closes #37!

### CODE REVIEW

File: backend/src/resources/setups.ts [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+  },
+  { setup: "What's a girl's best friend?", type: SetupType.pickOne },
+  {
+    setup:
+      "Dear Abby, I'm having some trouble with __________________ and would like your advice.",
+    type: SetupType.pickOne,
+  },
+  {
+    setup:
+      "In M. Night Shyamalan's new movie, Bruce Willis discovers that __________had really been _______________ all along.",
```
Comment (dikunong) [inline backend/src/resources/setups.ts:173]: Typo hehe

File: backend/src/resources/setups.ts [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+  { setup: "Why can't I sleep at night?", type: SetupType.pickOne },
+  { setup: "What's that smell?", type: SetupType.pickOne },
+  {
+    setup:
+      "After eight years in the White House, how is Obama finally letting loose?",
+    type: SetupType.pickOne,
+  },
+  {
+    setup:
+      "This is the way the world ends This is the way the world ends Not with a bang but with __________________.",
```
Comment (dikunong) [inline backend/src/resources/setups.ts:261]: How do we handle line breaks in setups?

**Review by dikunong — DISMISSED** [review]
That was a lot of data entry! Just a couple of things I noticed but nothing blocking

File: backend/src/resources/setups.ts (hunk 2 above)
Comment (taitfuller) [reply backend/src/resources/setups.ts]: This is a weird setup based on a poem - I've fixed formatting in 2a01133 by adding full stops

File: backend/src/resources/setups.ts (hunk 1 above)
Comment (taitfuller) [reply backend/src/resources/setups.ts]: Heheh 🤭

**Review by dikunong — APPROVED** [review]

**Pull request merged**